source/api/galaxy/Util.py

```diff
@@ -11,12 +12,11 @@
 class Util:
 
     @staticmethod
-    def check_parameters(yaml_path: str, output_dir: str, kwargs, location):
+    def check_parameters(yaml_path: Path, output_dir: Path, kwargs, location):
```

Comment:
is this supposed to be like this? asserting that it's the file with os.path?

Reply:
os.path works with path objects as well as string, but I can add an assert that it should be a Path object